project-base/tests/App/Test/FunctionalTestCase.php

```diff
@@ -73,10 +78,10 @@ protected function findClient(
         $kernelOptions = array_replace($defaultKernelOptions, $kernelOptions);
 
         if ($createNew) {
-            $this->client = $this->createClient($kernelOptions);
+            $this->client = $this->createClient($kernelOptions, ['HTTP_HOST' => 'webserver:8080']);
```

Comment:
This should use parameter

Reply:
It was fixed in e84403a3256d382762ed0c2a065188998e5b0b2e